components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityKeyStoreResolverConstants.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.core.util;
+
+/**
+ * This class holds the constants used by IdentityKeyStoreManager.
+ */
+public class IdentityKeyStoreResolverConstants {
+    
+    // CustomKeyStoreMapping config path
```

Comment:
Dot after end of the comment. Change in all places

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/5794/commits/6040b855d40daa0ff250b91a1930c5ac6a53ee6c